src/elements/ops.rs

```diff
@@ -1314,11 +1321,20 @@ impl Deserialize for Instruction {
 				I64REINTERPRETF64 => I64ReinterpretF64,
 				F32REINTERPRETI32 => F32ReinterpretI32,
 				F64REINTERPRETI64 => F64ReinterpretI64,
-				I32_EXTEND8_S => I32Extend8S,
-				I32_EXTEND16_S => I32Extend16S,
-				I64_EXTEND8_S => I64Extend8S,
-				I64_EXTEND16_S => I64Extend16S,
-				I64_EXTEND32_S => I64Extend32S,
+
+				#[cfg(feature="sign_ext")]
+				I32_EXTEND8_S |
```

Comment:
This seems weirder to me than just putting `#[cfg(feature="sign_ext")]` over each match arm.

Reply:
Both are OK, I think. I prefer this one since it is more consistent since every other conditional extension has their own arm.

Reply:
I don't feel strongly. Your call.